src/js/schemas/service-schema/Networking.js

```diff
@@ -39,9 +43,12 @@ const Networking = {
 
         return portMappings.map(function (portMapping) {
           return {
-            lbPort: portMapping.hostPort || portMapping.containerPort,
+            lbPort: portMapping.hostPort || portMapping.containerPort ||
```

Comment:
Not relevant for this PR, because it is working in this case, but this should have proper `null`-evaluation in the future, because a port `0` is treated as `false` in the logic. Could possibly lead to errors if a refactoring will be done at another place in the code.